### ISSUE TITLE
Riot-Desktop does not honor jitsi preferredDomain setting, Arch Linux

### ISSUE THREAD

**Issue by rajil** [vector-im/riot-web#13114]
Hello,
I created three separate Riot-Desktop profiles and started them using 'riot-desktop --profile-dir' option. For these three users, i set the jitsi preferred domain url using the following config,
```
{
    "jitsi": {
        "preferredDomain": "meet.mydomain.com"
    }
}

```
I checked the domain is being set by using Ctrl+Shif+I, and looking at mxReactSdkConfig.jitsi. It reported 'meet.mydomain.com' correctly for all sessions. This confirmed that the settings is being enforced.

I initiated the VideoCall using the 'video call' button which added the Jitsi widget. Once, I join the conference the Jitsi link shown is still pointing to 'jitsi.riot.im' rathe rather than 'meet.mydomain.com'

The riot-desktop version is 1.5.15.

Any idea why riot is not using my jitsi server?

**Comment by turt2live** [comment]
@rajil I can't reproduce this issue. Please try again, ensuring that the app is killed from the task tray, then go to Settings -> Help and Submit Debug Logs for this issue if it still doesn't work.

**Comment by turt2live** [comment]
This might an Arch specific problem as it should definitely be reading the config. There's no other option for it do anything: it has to read the config in order for the call buttons to work. 

OP in #riot-web provided screenshots of the Jitsi URL being random characters, indicating either it was created by an integration manager, is old pre-config change, or the Riot version is not accurate.

Needs more investigation - my Arch VM is broken.

**Comment by rajil** [comment]
'it should definitely be reading the config', if by this you mean config.json it is indeed reading that as mxReactSdkConfig.jitsi does report 'meet.mydomain.com' in Riot devtools.

**Comment by turt2live** [comment]
As per #riot-web, all the code and pieces in the process point to it working. The logs show a Scalar (integration manager) connection, but the log statements come from the code path which adds the new Jitsi widget. The logs also reveal that the code is using the new Jitsi widget construction to match the format. 

There should be no reason why it doesn't work. The logs even show a console.log of the config's `jitsi` section, which is accurate and error-free. 

Current suggestion is to wait for 1.15.16 and see if that fixes it as the code is different.

**Comment by rajil** [comment]
I installed riot-web-1.5.15 from the snap packages and pointed it to my config,
`$riot-web --profile-dir ~/.config/Riot`

This time around the jitsi widget did pickup my self-hosted Jitsi server. The jitsi widget correctly showed the url of my jitsi server. The camera refuses to work though, i get an error on making p2p video calls,
`Couldn't start capturing media! Is your microphone set up and does this app have permission?`

Bottom line the snap package is working fine with respect to Jitsi but the Arch Linux AUR package is not.

**Comment by turt2live** [comment]
The snap package not having microphone access sounds like a different bug. We don't maintain the snap package, so I would start with filing a bug with them.

The issue you've linked is unrelated to the arch package.

It looks like whatever version of the arch package you have is broken, and there's nothing we can do about it, sorry. I'm going to close this as it's not a bug we can fix - everything says it should be working fine and it works fine for everyone else at the moment.

Please try 1.15.16 when it is released.

**Comment by rajil** [comment]
Solved the mystery. Arch Linux has two packages riot-desktop and riot-web, the former depends on the latter. On my system even though riot-desktop was updated to 1.5.15, riot-web version was still 1.5.12. After updating riot-web, jitsi URL is working as expected.
